src/offer/offerService.js

```diff
@@ -7,4 +10,12 @@ export class OfferService {
     const offers = await this.offerRepository.getOffers();
     return offers;
   };
+
+  deleteOffer = async (id) => {
+    const isDelete = await this.offerRepository.deleteOffer(id);
+    if (isDelete.success !== true) {
+      throw new FailedToDeleteError(FAILED_DELETING, OFFERS);
+    }
+    return isDelete;
```

Comment:
It is not necessary to return anything, if it does not work it will throw an exception